x-pack/plugins/security_solution/public/management/pages/event_filters/store/middleware.ts

```diff
@@ -232,9 +232,9 @@ const refreshListDataIfNeeded: MiddlewareActionHandler = async (store, eventFilt
     dispatch({
       type: 'eventFiltersListPageDataChanged',
       payload: {
-        type: 'LoadingResourceState',
         // Ignore will be fixed with when AsyncResourceState is refactored (#830)
         // @ts-ignore
+        type: 'LoadingResourceState',
```

Comment:
I decided to add this type into the ts-ignore because I was getting this ts-error:
```
{ type: "LoadingResourceState"; previousState: ImmutableObject<UninitialisedResourceState> | ImmutableObject<LoadingResourceState<EventFiltersListPageData, ServerApiError>> | ImmutableObject<...> | ImmutableObject<...>; }' is not assignable to type 'ImmutableObject<{ forceRefresh: boolean; }>'.
        Object literal may only specify known properties, and 'type' does not exist in type 'ImmutableObject<{ forceRefresh: boolean; }>
        ```

Reply:
Yes, this is a known issue I think